AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -4451,7 +4451,7 @@ visitmons.be##.cookie-compliance-message
 sueddeutsche.de##.cookie-consent
 europe.nflshop.com,neuronation.com,conjuga-me.net,proz.com,htc.com,index.hr,vice.com##.cookie-consent-container
 theinquirer.net,v3.co.uk##.cookie-consent-content
-deloitte.com##.cookie-container
+yapikredi.com.tr,deloitte.com##.cookie-container
```

Comment:
Have you checked it with app? It won't help - the site is in the HTTPS exclusion list.